extensions/wikia/FlowTracking/FlowTracking.hooks.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+class FlowTrackingHooks {
+	const CREATE_PAGE_UNRECOGNIZED_FLOW = 'create-page-unrecognized';
+
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
+		\Wikia::addAssetsToOutput( 'flow_tracking_create_page_js' );
+
+		return true;
+	}
+
+	/**
+	 * @param  Page $page The created page object
+	 * @param  User $user The user who created the page
+	 * @param  string $text Text of the new article
+	 * @param  string $summary Edit summary
+	 * @param  int $minoredit Minor edit flag
+	 * @param  boolean $watchThis Whether or not the user should watch the page
+	 * @param  null $sectionAnchor Not used, set to null
+	 * @param  int $flags Flags for this page
+	 * @param  Revision $revision The newly inserted revision object
+	 * @return bool
+	 */
+	public static function onArticleInsertComplete( Page $page, User $user, $text, $summary, $minoredit,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$page, $user, $text, $summary, $minoredit, $watchThis, $sectionAnchor, $flags". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 9 parameters, which is greater than the 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS107)